### PR TITLE
[AN-7048] Add msg size limit

### PR DESCRIPTION
**Pull request by CalumMcCall**
### Issues

Currently Android allows the sending of any size text message, this PR adds a check which displays an error if the user attempts to send a message greater than 9000 chars in length. The jira ticket for this issue is [here](https://wearezeta.atlassian.net/browse/AN-7048).

### Testing

Testing was done manually.
#### APK
[Download build #2604](http://10.10.124.11:8080/job/Pull%20Request%20Builder/2604/artifact/build/artifact/wire-dev-PR3001-2604.apk)
[Download build #2608](http://10.10.124.11:8080/job/Pull%20Request%20Builder/2608/artifact/build/artifact/wire-dev-PR3001-2608.apk)

### CODE REVIEW

**Comment by AndroidBob** [comment]
Can one of the admins verify this patch?

**Review by BradleyWire — APPROVED** [review]

**Comment by BradleyWire** [comment]
ok to test

File: app/src/main/scala/com/waz/zclient/cursor/CursorController.scala [hunk 1]
```diff
@@ -219,36 +219,50 @@ class CursorController(implicit inj: Injector, ctx: Context, evc: EventContext)
...
-
-              Future {
-                msgBeingSendInConv.mutate(_ - msg.convId)
+  private def checkMsgLength(msg: String): Boolean = {
+    val MAX_MSG_LENGTH = 9000
+    if (msg.length > MAX_MSG_LENGTH) {
+      new AlertDialog.Builder(activity)
+        .setTitle(getString(R.string.conversation_input_bar_message_too_long_title))
+        .setMessage(getString(R.string.conversation_input_bar_message_too_long_message, MAX_MSG_LENGTH.toString))
+        .setPositiveButton(android.R.string.ok, null)
```
Comment (gizemb) [inline app/src/main/scala/com/waz/zclient/cursor/CursorController.scala:228]: i suppose we shouldn't use `android.R` resources as they're not translated.

**Review by gizemb — CHANGES_REQUESTED** [review]

File: app/src/main/scala/com/waz/zclient/cursor/CursorController.scala (hunk 1 above)
Comment (CalumMcCall) [reply app/src/main/scala/com/waz/zclient/cursor/CursorController.scala]: Good catch!

**Review by gizemb — APPROVED** [review]

**Pull request merged**